package.json

```diff
@@ -29,7 +29,8 @@
     "lint": "prettier --list-different \"packages/*/src/**/*.{js,jsx,ts,tsx}\"",
     "lint-fix": "prettier --write \"packages/*/src/**/*.{js,jsx,ts,tsx}\"",
     "posttest": "npm run lint",
-    "release": "npm run clean && npm ci && lerna publish --exact",
+    "release:version-bump": "lerna version --exact -m 'Release'",
+    "release:start-ci-publish": "node -p '`Publish (dist-tag:${process.env.APOLLO_DIST_TAG || \"latest\"})`' | git tag -F - \"publish/$(date -u '+%Y%m%d%H%M%S')\" && git push origin \"$(git describe --match='publish/*' --tags --exact-match HEAD)\"",
```

Comment:
Note the scripts are named differently, but they behave a bit differently, so it seemed worth not re-using the same name.